src/R/Components/Impl/PackageManager/Implementation/ViewModel/RPackageManagerViewModel.cs

```diff
@@ -575,28 +560,17 @@ internal class RPackageManagerViewModel : BindableBase, IRPackageManagerViewMode
         }
 
         public async Task SwitchToLoadedPackagesAsync() {
-            await _coreShell.SwitchToMainThreadAsync();
-            if (_selectedTab == SelectedTab.LoadedPackages) {
-                return;
-            }
-
-            _selectedTab = SelectedTab.LoadedPackages;
-            if (_installedAndLoadedLock.IsCompleted) {
-                ReplaceItems(_loadedPackages);
-                return;
-            }
-
-            IsLoading = true;
-
-            await ReloadInstalledAndLoadedPackagesAsync();
-            if (_selectedTab == SelectedTab.LoadedPackages) {
-                IsLoading = false;
-                ReplaceItems(_loadedPackages);
+            if (await SetTabAsync(SelectedTab.LoadedPackages)) {
```

Comment:
>SetTabAsync [](start = 22, length = 11)

Please add `await _coreShell.SwitchToMainThreadAsync();` at the beginning of this method